engine/src/execution_graph/logic_controllers/LogicalProject.cpp

```diff
@@ -17,6 +17,7 @@
 #include "../../Interpreter/interpreter_cpp.h"
 #include "../../parser/expression_tree.hpp"
 #include "../../Utils.cuh"
+#include "utilities/DebuggingUtils.h"
```

Comment:
this is probable left over from debugging im guessing :)

Reply:
yep removed thanks